nx.json

```diff
@@ -0,0 +1,29 @@
+{
+  "tasksRunnerOptions": {
+    "default": {
+      "runner": "nx/tasks-runners/default",
+      "options": {
+        "cacheableOperations": [
+          "dist",
+          "build",
+          "test",
+          "lint",
+          "test:ci",
+          "tsc"
```

Comment:
Where do these operations come from? They don't match what we have within this repo.

Reply:
This is the set of "default" commands which should be adjusted to your needs and situation. We did not want to just populate the list with all your available script commands to avoid unexpected behavior.

Reply:
Okay, I'll update the file then.
Thanks